seerep-hdf5/seerep-hdf5-fb/src/hdf5-fb-pointcloud.cpp

```diff
@@ -0,0 +1,814 @@
+#include "seerep-hdf5-fb/hdf5-fb-pointcloud.h"
+
+#include <highfive/H5DataSet.hpp>
+
+namespace seerep_hdf5_fb
+{
+Hdf5FbPointCloud::Hdf5FbPointCloud(std::shared_ptr<HighFive::File>& file, std::shared_ptr<std::mutex>& writeMtx)
+  : Hdf5FbGeneral(file, writeMtx)
+{
+}
+
+// TODO refactor again, still not happy with the API
+
+void Hdf5FbPointCloud::writePointCloud2(const std::string& id, const seerep::fb::PointCloud2& cloud,
```

Comment:
Methodes are too long in this file. Please break them down in multiple methodes.

Reply:
The methods should now have a reasonable length, except:

https://github.com/agri-gaia/seerep/blob/4524908f6db045517f4509cbca945f0e7793d217/seerep-hdf5/seerep-hdf5-fb/src/hdf5-fb-pointcloud.cpp#L201-L320

But it's hard to break the method down without using >8 parameters, is it sill  worth it?